inlang/source-code/paraglide/paraglide-js-plugin/src/ideExtension/messageReferenceMatchers.test.ts

```diff
@@ -0,0 +1,163 @@
+import { describe, expect, it } from "vitest"
+import { parse } from "./messageReferenceMatchers.js" // Replace with the actual filename
+
+describe("Paraglide Message Parser", () => {
+	it("should match simple m function call", () => {
+		const sourceCode = `
+		import * as m from "@inlang/paraglide-js/example/messages";
```

Comment:
please add a test for different namespaces. 

```ts
import * as m from "@inlang/paraglide-js/{namespace}/messages"
```

as well as no namespace (which I am going to allow to simplify the getting started https://github.com/inlang/monorepo/discussions/1464#discussioncomment-7280355)

```ts
import * as m from "@inlang/paraglide-js/messages"
```

Reply:
Solved 👍